assets/errata.js

```diff
@@ -13,115 +13,160 @@ a new comment on the issue may be added beginning with the word "Summary".
 As for the report, the structure of the HTML file can be seen in the index.html file. The net result is
 that the active issues are displayed in different sections, depending on the presence of specific labels. The relevant values are set
 through some data-* attributes on the elements.
+*/
 
-TODO: This code should be re-written...
+/**
+ * Fetch a JSON file.
+ * 
+ * @async
+ * @param {string} resource_url 
+ */
+async function fetch_json(resource_url) {
```

Comment:
How about using [json()](https://fetch.spec.whatwg.org/#dom-body-json) instead
`async function fetch_json(resource_url) {
  return  fetch(resource_url).then(response => response.json());
}`

Reply:
Oops, I missed this one; did not realize it has this built-in feature. Neat.